src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/ViewModel/ProjectHomeViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome.ViewModel
+
+import org.wycliffeassociates.otter.common.data.dao.Dao
+import org.wycliffeassociates.otter.common.data.model.Project
+import org.wycliffeassociates.otter.common.domain.GetProjectsUseCase
+import tornadofx.*
+import org.wycliffeassociates.otter.common.data.model.Language
+import io.reactivex.Observable
+import java.util.*
+
+class ProjectHomeViewModel: ViewModel() {
+    // TODO(need repo)?
+    val getProjectsUseCase = GetProjectsUseCase()
```

Comment:
the GetProjectsUserCase has a Dao<Project> as a parameter. I'm drawing a blank as to what to do here... 
or do I end up injecting the usecase in the first place?